src/navigator-card-body.jsx

```diff
@@ -412,10 +349,92 @@ export const NavigatorCardBody = ({
                   className="pf-m-no-border-rows"
                   variant="compact"
                   columns={[_("Name")]}
-                  rows={sortedFiles.map(file => ({ columns: [{ title: <Item file={file} key={file.name} /> }] }))}
+                  rows={sortedFiles.map(file => ({
+                      columns: [
+                          {
+                              title:
+    <Item
+      file={file} key={file.name}
+      selected={selected}
+      setSelected={setSelected}
+      onDoubleClickNavigate={onDoubleClickNavigate}
+      isGrid={isGrid}
+    />
```

Comment:
The indentation is a bit weird here.

Reply:
That's how eslint was happy, it doesn't automatically fix it. And I couldn't find any other way.

Reply:
I think you need to manually massage it, but I'd have to take a look. Anyway this is already a much better overall improvement